app/views/games/show.html.erb

```diff
@@ -1,32 +1,37 @@
 <div class ="game-show">
    <div>
    <div id="home-slideshow" class="carousel slide" data-ride="carousel">
-     <ol class="carousel-indicators">
-
-       <% @game.pictures.each_with_index do |pic, i| %>
-       <li data-target="#home-slideshow" data-slide-to= <%= "#{i}" %>
-         <% if i == 0 %>
-          class="active">
-         <% else %>
-          class="">
+     <% if @game.pictures.length.zero? %>
+      <div class="carousel-inner" role="listbox">
+        <%= image_tag "fallback/default.png", class: "carousel-image" %>
+      </div>
+     <% else %>
+       <ol class="carousel-indicators">
+         <% @game.pictures.each_with_index do |pic, i| %>
+           <li data-target="#home-slideshow" data-slide-to= <%= "#{i}" %>
+             <% if i == 0 %>
+             class="active">
+             <% else %>
+             class="">
+             <% end %>
+           </li>
          <% end %>
-       </li>
-       <% end %>
-     </ol>
-     <div class="carousel-inner" role="listbox">
-       <% @game.pictures.each_with_index do |picture, index| %>
-        <% url = picture.url %>
-        <% if  index == 0 %>
+       </ol>
+       <div class="carousel-inner" role="listbox">
+         <% @game.pictures.each_with_index do |picture, index| %>
+         <% url = picture.url %>
+         <% if  index == 0 %>
          <div class="item active">
            <%= image_tag url, class: "carousel-image" %>
          </div>
-        <% else  %>
+         <% else  %>
          <div class="item">
            <%= image_tag url, class: "carousel-image" %>
          </div>
-        <% end %>
-       <% end %>
-     </div>
+         <% end %>
```

Comment:
Two `end` at the same indent level. That's confusing. You probably have some indenting to fix here